kvmagent/kvmagent/plugins/vm_metadata/rw.py

```
@@ -0,0 +1,738 @@
+    while offset < lv_size:
+        if time.time() > deadline:
+            logger.warning("Brute-force scan timed out after %ds at "
+                           "offset %d / %d",
+                           BRUTE_FORCE_TIMEOUT_SEC, offset, lv_size)
+            break
+        chunk = min(BRUTE_FORCE_CHUNK_SIZE, lv_size - offset)
+        if chunk < ALIGNMENT:
+            break
+        try:
+            data = aligned_pread(fd, chunk, offset)
+        except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)